### PR TITLE
added initial code - PR

### PR DESCRIPTION
**Pull request by ericaig**
hello world pull request

### CODE REVIEW

File: staff/bamidele-aighewi/hello-world/index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Hello World!</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>Hello, World!</h1>
```
Comment (ericaig) [inline staff/bamidele-aighewi/hello-world/index.html]: h1: blah blah blah

File: staff/bamidele-aighewi/hello-world/style.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+body{
+    background-color: yellow;
+}
+
+h1{
+    cursor: pointer;
```
Comment (ericaig) [inline staff/bamidele-aighewi/hello-world/style.css]: cursor: bla

File: staff/bamidele-aighewi/hello-world/index.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Hello World!</title>
```
Comment (victoriCrediday) [inline staff/bamidele-aighewi/hello-world/index.html]: Exc!

File: staff/bamidele-aighewi/hello-world/index.html [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>Hello, World!</h1>
+    <p>ykSlab coders Academiiii.</p>
```
Comment (victoriCrediday) [inline staff/bamidele-aighewi/hello-world/index.html]: this text should be `<p>Skylab Coders Academy</p>`

**Pull request merged**